### DISCUSSION TITLE
Stability of Euler Bernoulli Explicit Euler Scheme

### DISCUSSION THREAD

**Discussion by Julian-Wyatt** [wenceorg/comp4187#57] [Q&A] [answered]
As per question 4 of the coursework, do we have to prove the question analytically or by experimentation to ensure stability?

**Comment by wence-** [comment]
I think (I hope @annereinarz will correct me if I am wrong) that you have seen the necessary tools to check Q4 analytically (there are still some numerics involved because you need to pick a particular grid spacing to get the spatial operator). That said, I would also do a brief numerical experiment to confirm the "pen-and-paper" analysis.

**Comment by annereinarz** [comment, marked as answer]
We have covered the stability radius of explicit Euler, but we haven't yet had a look at this in the context of spatial discretisations. The question is in the coursework to remind you to check that the time-step you have chosen leads to a stable scheme. So I would say that it is fully sufficient to check experimentally.

**Reply by Julian-Wyatt** [reply]
Thanks very much both!
